tests/end2end/features/misc/clipboard.feature

```diff
@@ -26,3 +26,28 @@ Feature: Interaction with the system clipboard.
         When I run copy-name
         And I run paste-name
         Then the working directory should be child_01
+
+    Scenario: Copy image to clipboard.
+        Given I open the image '.hidden.jpg'
```

Comment:
This `Given` step allows defining the name of the image we open, so you can use any name you like. If you don't need a specific name, you can just use `Given I open any image` which will create `image.jpg`.